sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -232,6 +232,9 @@ primaryExpression
     | name=CURRENT_TIME ('(' precision=integerLiteral')')?                           #specialDateTimeFunction
     | name=CURRENT_TIMESTAMP ('(' precision=integerLiteral')')?                      #specialDateTimeFunction
     | CURRENT_SCHEMA ('(' ')')?                                                      #currentSchema
+    | CURRENT_USER                                                                   #currentUser
+    | USER                                                                           #user
```

Comment:
if ``USER`` is always returning ``CURRENT_USER`` they can be merged in the same statement ```   | (CURRENT_USER | USER)                                                          #currentUser``` and `visitUser ` won't be needed